app/models/spree/payment_decorator.rb

```diff
@@ -2,116 +2,9 @@
 
 module Spree
   Payment.class_eval do
```

Comment:
we have to delete the decorator file, right?

Reply:
That persist invalid bugs me and I want to understand what consequences can that override have.